Tests/Foundation/main.swift

```diff
@@ -124,6 +124,7 @@ var allTestCases = [
     testCase(TestUnitVolume.allTests),
     testCase(TestNSLock.allTests),
     testCase(TestNSSortDescriptor.allTests),
+    testCase(TestFileHandleAsync.allTests),
```

Comment:
(Let me leave a comment before the test logs would gone.)
Build failure here:
`error: invalid conversion from 'async' function of type '() async throws -> ()' to synchronous function type '() throws -> Void'`